src/csharp/akka_validation/Network.cs

```diff
@@ -179,19 +182,26 @@ protected override void OnReceive(object message)
                 var sender = broadcast.Sender;
                 var broadcasted = broadcast.Message;
                 var topic = broadcasted.GetType();
+                var msgId = _nextMessageId++;
```

Comment:
We use the same message id for all tuples, because that is the way omnet expects it and the way omnet will trigger events based on that broadcast and push them back to us.